packages/ignite-cli/src/extensions/ignite/addModule.js

```diff
@@ -23,10 +23,14 @@ module.exports = (plugin, command, context) => {
     // install the module
     if (useYarn) {
       const addSwitch = options.dev ? '--dev' : ''
-      await system.run(`yarn add ${moduleFullName} ${addSwitch}`)
+      const cmd = `yarn add ${moduleFullName} ${addSwitch}`
+      ignite.log(cmd)
+      await system.spawn(cmd, { stdio: 'ignore' })
```

Comment:
maybe we can wrap `spawn` with {stdio: 'ignore} and call it `runSafe`.  Perhaps make it the prescribed way to run commands in gluegun, with a short explanation why?  I could see this being an issue for anyone who uses gluegun to use the wrong path.